.travis.yml

```diff
@@ -7,12 +7,17 @@ env:
     - PIPENV_IGNORE_VIRTUALENVS=1
 before_install:
   - pip install pipenv
+  - docker pull readthedocs/build:master || true
 install:
   - pipenv install
 script:
-  - pipenv run pytest --suppress-no-test-exit-code
-  - docker build .
+  - docker build --cache-from readthedocs/build:master -t readthedocs/build:master .
```

Comment:
looks like we don't the context, you can pass the dockerfile as `- < Dockerfile`